modules/services/polybar.nix

```diff
@@ -103,6 +105,27 @@ in {
         '';
         example = "polybar bar &";
       };
+
+      extraEnv = mkOption {
+        type = types.attrsOf types.str;
+        description = ''
+          Any extra environment variables to be set for the service.
+        '';
+        default = { };
+        example = {
+          DISPLAY = ":0";
```

Comment:
The `DISPLAY` variable is already imported into the systemd session by default when using HM X session management:
https://github.com/nix-community/home-manager/blob/b6ed605d4a5a75fae25132e4f1da15f3fd635bc1/modules/xsession.nix#L78-L86

If not using X session management then I think it it would be better to do `systemctl --user import-environment DISPLAY` in your `~/.xprofile`.

Reply:
Got it, this was just an example anyway :+1: